pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -502,7 +502,11 @@ def paint(self, p, opt, widget):
             p.setPen(sp)
             p.drawPath(path)
         p.setPen(cp)
-        p.drawPath(path)
+        if self.fillPath is not None:
+            p.drawPath(self.fillPath)
+        else:
+            p.drawPath(path)
+        profiler('drawPath')
```

Comment:
Mind removing the duplicate call to `profiler('drawPath')`?

Reply:
Done.